### ISSUE TITLE
More apps support

### ISSUE THREAD

**Issue by markedone66** [AKruimink/WinReform#48]
Trying to resize a game window but it doesnt even appear in the process list :( would it be possible to somehow add support to link an EXE file so itwould monitorwhen it starts up so it has easier time to find the running process ?

or this app is intended for very primitive window types?

**Comment by AKruimink** [comment]
The app gets all active processes that that are running on the system. 
However, the processes it can obtain are limited to the access level of WinReform. 

In the latest [release](https://github.com/AKruimink/WinReform/releases/tag/v2.0.0), the requirement to run as administrator was removed, as was discussed in #40. this means that if your looking for a process that runs with elevated privileges, Winreform should also be ran as administrator to see them.

Also, sometimes the process that own the window, isn't the same process of the application with the icon, there might be cases where some obscure process needs to be found.

If the suggestion above doesn't resolve the issue, could you please provide some additional infrormation, like the app your trying to resize etc

**Comment by Fizub** [comment]
Yeah, its a 32 bit application. You will have to build it yourself for now bro
